### PR TITLE
Add env var to not replace systemd service file

### PR DESCRIPTION
**Pull request by suchmememanyskill**
Please tick as appropriate:
- [X] I have tested this code on a steam deck or on a PC
- [X] My changes generate no new errors/warnings
- [ ] This is a bugfix/hotfix
- [X] This is a new feature

# Description

This adds an environment variable that stops decky from updating it's systemd unit/config, so any configuration in it will not be lost. Default behaviour stays the same: the systemd unit/config gets updated every time decky updates

Tl;dr: Simply add `Environment=KEEP_SYSTEMD_SERVICE=1` to decky's systemd unit/config to stop it from being overwritten

PS: This also fixes a bug where the LIVE_RELOAD environment variable was ineffective

### CODE REVIEW

**Review by TrainDoctor — APPROVED** [review]
Tested, working. Merging.
